Packs/SpamhausFeed/Integrations/SpamhausFeed/SpamhausFeed.yml

```diff
@@ -0,0 +1,137 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: SpamhausFeed
+  version: -1
+configuration:
+- display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- defaultvalue: '240'
+  display: Fetch Interval
+  name: feedFetchInterval
+  required: true
+  type: 19
+- additionalinfo: Type of the indicator in the feed.
+  display: Indicator Type
+  name: indicator_type
+  required: true
+  type: 0
```

Comment:
It's kindof basa that we need to include this basically for only in the case of the generic HTTP Feed integration... whereas every specific implementation that uses the http module should in-theory have the indicator type hardcoded as part of the feed_type dictionary...